packages/backend/src/config/app.ts

```diff
@@ -17,10 +17,10 @@ function envFloat(name: string, value: number): number {
   return envValue == null ? value : +envValue
 }
 
-// function envBool(name: string, value: boolean): boolean {
-//   const envValue = process.env[name]
-//   return envValue == null ? value : Boolean(envValue)
-// }
+function envBool(name: string, value: boolean): boolean {
+  const envValue = process.env[name]
+  return envValue == null ? value : Boolean(envValue)
```

Comment:
```suggestion
  return envValue == null ? value : envValue === 'true'
```
https://github.com/interledger/rafiki/pull/588#discussion_r966041927
https://github.com/interledger/rafiki/blob/e796dedeee7c3aedd43c5c977ce83f420ea8667c/packages/auth/src/config/app.ts#L13-L16